### DISCUSSION TITLE
【灌水】

### DISCUSSION THREAD

**Discussion by wugan-bot156** [share121/inter-knot#1142] [其他]
# 就是说这个绳网可不可以和米哈游交流一下，反正在光映广场zzz工作室旁边的乙都说可以把任何好玩儿的事情都告诉他们了 。~~不行就和他们爆了~~ 
# $\color{#6495ED}{当然了，只是要点小建议，不是说公开化}$ :ghost:
![无标题5_20230316214410(1)](https://github.com/user-attachments/assets/e028ad92-e597-45ce-b120-128b8b802fa5)

**Comment by pomark20** [comment]
有道理，到时候打开米游社进去的ZZZ进去就是这个界面挺好的。

**Comment by chaochao6667** [comment]
那样门槛会被降低吗？害怕赛博生命和xz闻着味就来了

**Comment by YukariZZZ** [comment]
确实，难得的净土害怕被污染了。

**Comment by YukariZZZ** [comment]
建议米游社一个表绳网，这里一个里绳网

**Comment by AnbulesPeo** [comment]
还是门槛高一点比较好吧，不希望太大众化

**Comment by GanlinSama** [comment]
支持，还是不要太大众化了好，门槛高一点论坛就干净一点

**Comment by SaKura-071721** [comment]
人少归少，但是没有魔怔人。这样就挺好了

**Comment by Soldier11-ObsidianBarracks** [comment]
天才

**Comment by wugan-bot156** [comment]
为啥和preview里显示的不一样，我的蓝色字体咋不显示

**Reply by lllliufeng** [reply]
编辑界面可以吧，绳网里面不行，跟数学公式一样
$$\left( \sum_{k=1}^n a_k b_k \right)^2 \leq \left( \sum_{k=1}^n a_k^2 \right) \left( \sum_{k=1}^n b_k^2 \right)$$

**Comment by szlx2333** [comment]
y1s1 绳网应该是少数人知道且不能在明面上说的东西

**Comment by GenshinOnly** [comment]
任何出现在绝区零里的内容米家都要负责的，除非对所有内容都人工审核，不然应该不可能

**Comment by dominic121-dot** [comment]
你说得对，但剧情里感觉是个人就能在绳网上发委托，不过毕竟末世背景官方对绳网这种管的松也正常

**Comment by dengyunzhe** [comment]
绳网可是非法网络，不能让官方看到的（bushi）
